x/group/client/testutil/tx.go

```diff
@@ -2347,3 +2530,155 @@ func (s *IntegrationTestSuite) createCLIProposal(groupPolicyAddress, proposer, s
 
 	return testutil.WriteToNewTempFile(s.T(), string(bz)).Name()
 }
+
+func (s *IntegrationTestSuite) createAccounts(quantity int) []string {
+	val := s.network.Validators[0]
+	clientCtx := val.ClientCtx
+	accounts := make([]string, quantity)
+	commonFlags := []string{
+		fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+		fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+	}
+
+	for i := 1; i <= quantity; i++ {
+		memberNumber := uuid.New().String()
+
+		info, _, err := clientCtx.Keyring.NewMnemonic(fmt.Sprintf("member%s", memberNumber), keyring.English, sdk.FullFundraiserPath,
+			keyring.DefaultBIP39Passphrase, hd.Secp256k1)
+		s.Require().NoError(err)
+		//s.nextAccount++
+
+		pk, err := info.GetPubKey()
+		s.Require().NoError(err)
+
+		account := sdk.AccAddress(pk.Address())
+		accounts[i-1] = account.String()
+
+		_, err = banktestutil.MsgSendExec(clientCtx, val.Address, account,
+			sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(1000))),
+			commonFlags...,
+		)
+		s.Require().NoError(err)
+	}
+	return accounts
+}
+
+func (s *IntegrationTestSuite) createGroupWithMembers(membersWeight, membersAddress []string) string {
+	s.Require().Equal(len(membersWeight), len(membersAddress))
+	val := s.network.Validators[0]
+	clientCtx := val.ClientCtx
+	commonFlags := []string{
+		fmt.Sprintf("--%s=true", flags.FlagSkipConfirmation),
+		fmt.Sprintf("--%s=%s", flags.FlagBroadcastMode, flags.BroadcastBlock),
+		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(s.cfg.BondDenom, sdk.NewInt(10))).String()),
+	}
+
+	membersValid := validMembers{}
+
+	for i, address := range membersAddress {
+		membersValid.Members = append(membersValid.Members, member{
+			address,
+			membersWeight[i],
+			validMetadata,
+		})
+	}
+	validMemberByte, err := json.MarshalIndent(&membersValid, "", "\t")
+	s.Require().NoError(err)
+	validMemberString := string(validMemberByte)
+	s.Require().NoError(err)
+	validMembersFile := testutil.WriteToNewTempFile(s.T(), validMemberString)
+	out, err := cli.ExecTestCLICmd(clientCtx, client.MsgCreateGroupCmd(),
+		append(
+			[]string{
+				val.Address.String(),
+				validMetadata,
+				validMembersFile.Name(),
+			},
+			commonFlags...,
+		),
+	)
+	s.Require().NoError(err, out.String())
+	var txResp sdk.TxResponse
+	s.Require().NoError(val.ClientCtx.Codec.UnmarshalJSON(out.Bytes(), &txResp), out.String())
+	return s.getGroupIdFromTxResponse(txResp)
+}
+
+func (s *IntegrationTestSuite) createGroupThresholdPolicyWithTokens(groupID string, threshold int, tokens int64) string {
+	val := s.network.Validators[0]
+	clientCtx := val.ClientCtx
+
+	commonFlags := []string{
```

Comment:
can we maybe define `commonFlags` as a module-level variable? It seems like the same definition all over this file

Reply:
you are right!